requirements.txt

```diff
@@ -5,3 +5,4 @@ pbr
 termcolor==1.1.0
 setproctitle
 ordereddict==1.1
+ujson
```

Comment:
Pip may not be available unfortunately that's why we want proper packaging. It's kinda logic to not have pip in production because most of time you have to compile something when installing from pip and gcc is not really recommended is production environments. You also need python.h (python-devel) which is not usually installed.

Another huge con, pip is not made to install configuration files and init script , it only install alignak as a lib, config file shipped are usually not in system path but those are "demo".